### PR TITLE
Extend TomlNumpyEncoder to numpy integer types.

### PR DESCRIPTION
**Pull request by filippo82**
I have extend TomlNumpyEncoder to handle numpy integer types. I've also added the unit test for these cases.

### CODE REVIEW

File: toml/encoder.py [hunk 1]
```diff
@@ -118,6 +118,10 @@ def _dump_float(v):
     return "{}".format(v).replace("e+0", "e+").replace("e-0", "e-")
 
 
+def _dump_int(v):
```
Comment (uiri) [inline toml/encoder.py]: Thank you for your pull request!

Could you move `_dump_int` to be inside of `TomlNumpyEncoder`?
I do not think there is any need to modify the existing encoder to extend the Numpy one to support integers.

Comment (filippo82) [reply toml/encoder.py]: Hi Will,

just to clarify, you are suggesting to revert `int: _dump_int` to `int: lambda v: v` inside the `TomlEncoder` class and to move the `_dump_int` function inside the `TomlNumpyEncode` class?

Cheers.

Comment (uiri) [reply toml/encoder.py]: Yes, that is my suggestion exactly :smiley:

Comment (filippo82) [reply toml/encoder.py]: Done!

**Pull request merged**